src/Cameras/VR/babylon.webVRCamera.ts

```diff
@@ -194,11 +196,11 @@ module BABYLON {
         } 
 
         public _checkInputs(): void {
-            if (this._vrEnabled) {
+            if (this._vrDevice && this._vrDevice.isPresenting) {
                 if (this._specsVersion === 1.1) {
                     this._vrDevice.getFrameData(this._frameData);
                 } else {
-                    //backwards comp
+                    // TODO: Does backwards comp need to be here any more? The Engine class doesn't support it any more.

```

Comment:
We support WebVR 1.0 only because of the GearVR browser. It was added after being completely removed. If the GearVR now supports 1.1 (I can't seem to find the information), we don't need 1.0 . if it doesn't, we need the full support.

Same goes to the changes in the engine.

Reply:
The Samsung browser on GearVR was upgraded to 1.1 a little while back now, so potentially safe to remove this. I propose full removal of 1.0 support in a subsequent PR to make things easier?

Reply:
Yes, good idea, let's try to split things. I wasn't sure for GearVR neither and wasn't able to had confirmation after looking on the web.

Reply:
@RaananW  : Fancy doing the other PR to remove webvr 1.0?

Reply:
@deltakosh - will do.

Reply:
Thanks!